src/components/QueryBuilder.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div class="querybuilder">
+		<h1 class="querybuilder__heading">Simple Query Builder</h1>
+		<p class="querybuilder__description">
+			Short explanatory text that also manages expectations. Short explanatory text that also
+			manages expectations.
+		</p>
+		<h4 class="querybuilder__find-title">Find all items...</h4>
```

Comment:
This should be an `<h2>` even if the h4 tokens are used, I think.

Reply:
In the figma there's nothing and only it's the css
![image](https://user-images.githubusercontent.com/5351225/94714504-834a1400-034c-11eb-8b69-bb7701d650cb.png)

I'm fine either way, maybe we should double check with the designers?

Reply:
I think this should only apply to the styling. As far as semantic markup goes the recommendation is pretty clear:

> Avoid skipping heading levels: always start from `<h1>`, next use `<h2>` and so on.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
oh thanks then. I fix it.